code/drasil-code/Data/Drasil/ExternalLibraries/ODELibraries.hs

```diff
@@ -293,15 +293,15 @@ solveFromToStep = quantfunc $ implVar "SolveFromToStep_oslo" (nounPhrase
   solT (label "SolveFromToStep")
 
 vecDepVar :: ODEInfo -> CodeVarChunk
-vecDepVar info = quantvar $ implVar (dv ^. uid) (dv ^. term) vecT
+vecDepVar info = quantvar $ implVar (showUID dv) (dv ^. term) vecT
   (sub (symbol dv Implementation) (label "vec"))
   where dv = depVar info
 
 -- Hack required because 
 -- | Oslo's Vector type behaves like an array, so needs to
 -- be represented as one or else will hit type errors in GOOL.
 arrayVecDepVar :: ODEInfo -> CodeVarChunk
-arrayVecDepVar info = quantvar $ implVar (dv ^. uid ++ "vec") (dv ^. term)
+arrayVecDepVar info = quantvar $ implVar (showUID dv ++ "vec") (dv ^. term)
```

Comment:
actually, should use `+++` here, then `showUID`. This will future-proof the code.

Reply:
Would we want a variant of `implVar` and `implVar'` that take in a `UID` rather than a `String`?

Reply:
We should. Then we can see if `implVar` is used at all in the remainders.

Reply:
I originally didn't want to use `+++` for this since it doesn't really show a direct dependency from one chunk to the next, but it should be fine